docs/layers/grid-layer.md

```diff
@@ -218,6 +218,26 @@ You should pass in the function defined outside the render function so it doesn'
  }
 ```
 
+
+##### `getColorWeight` (Function, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `point => 1`
+
+`getColorWeight` is the accessor function to get the weight of a point used to calcuate the color value for a cell.
+It takes the data prop array element and returns the weight, for example, to use `SPACE` field, `getColorWeight` should be set to `point => point.SPACES`.
```

Comment:
Move this to the section below, where you explain that these two props must be used together, and provide a few examples.

Ex 1. Use color to encode the total count of points:

```js
colorAggregation: 'SUM',
getColorWeight: point => 1
```

is equivalent to:

```js
getColorValue: points => points.length
```

Ex 2. Use color to encode the average number of spaces:

```js
colorAggregation: 'MEAN',
getColorWeight: point => point.spaces
```

is equivalent to:

```js
getColorValue: points => points.reduce((sum, p) => sum += p.spaces, 0) / points.length
```

Reply:
Added examples, will make same changes to `HexagonLayer` docs, in Part6 PR.